pubsub/test/iam.test.js

```diff
@@ -14,159 +14,278 @@
 'use strict';
 
 var proxyquire = require('proxyquire').noCallThru();
+var topicName = 'foo';
+var subscriptionName = 'bar';
+var policyMock = 'policy';
+var permissionsMock = 'permissions';
+
+function getSample () {
+  var subscriptionMock = {
```

Comment:
Any reason why **subscriptionMock** and **topicMock** are the same thing? Is there some way to create a copy or something other than writing the whole thing out again?